observability/logging/logging-6.0/log6x-output.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-output-types"]
+= Log output types
+:context: logging-output-types
+
+toc::[]
+
+Outputs define the destination where logs are sent to from a log forwarder. You can configure multiple types of outputs in the `ClusterLogForwarder` custom resource (CR) to send logs to servers that support different protocols.
+
+include::modules/log6x-supported-log-outputs.adoc[leveloffset=+1]
+
+[id="log6x-output-types-descriptions"]
+== Output type descriptions
+
+`default`:: The on-cluster, Red{nbsp}Hat managed log store. You are not required to configure the default output.
++
+[NOTE]
+====
+If you configure a `default` output, you receive an error message, because the `default` output name is reserved for referencing the on-cluster, Red{nbsp}Hat managed log store.
```

Comment:
As said, this is not valid for Logging 6.0 as the word "default" is not more valid, this phrase is only valid for Logging 5.y

Reply:
I will remove this Note.